### ISSUE TITLE
Adding a budget just overrides the existing budget

### ISSUE THREAD

**Issue by torrentails** [asafonov/monly.apk#8]
I can only have one budget object, as hitting `add budget` just overwrites and completely resets the existing one, rather than adding a new one

Android ver: 14.0.1
App version: 2.13

**Comment by torrentails** [comment]
Seems to only be the Groceries category that gets overwritten as it's hardcoded to the code:

[https://github.com/asafonov/monly.apk/blob/fb56f5f7a22c5c88dd5abf77cbc214358f2ef9f9/app/src/main/assets/main.js#L684](/app/src/main/assets/main.js#L684)

**Comment by asafonov** [comment]
Yes, I can reproduce it, will be fixed, thanks for pointing that out
